activemodel/lib/active_model/naming.rb

```diff
@@ -195,28 +195,38 @@ def initialize(klass, namespace = nil, name = nil, locale = :en)
     #
     # Specify +options+ with additional translating options.
     def human(options = {})
-      return @human unless @klass.respond_to?(:lookup_ancestors) &&
-                           @klass.respond_to?(:i18n_scope)
-
-      defaults = @klass.lookup_ancestors.map do |klass|
-        klass.model_name.i18n_key
-      end
+      return @human unless i18n_keys.present?
 
+      key, *defaults = i18n_keys
       defaults << options[:default] if options[:default]
-      defaults << @human
+      defaults << MISSING_TRANSLATION
 
-      options = { scope: [@klass.i18n_scope, :models], count: 1, default: defaults }.merge!(options.except(:default))
-      I18n.translate(defaults.shift, **options)
+      translation = I18n.translate(key, scope: i18n_scope, count: 1, **options, default: defaults)
+      translation == MISSING_TRANSLATION ? @human : translation
```

Comment:
I see you had this logic in another PR as well, perhaps we should make this a method that can take in args instead of repeating logic?